Havit.Blazor.Components.Web.Bootstrap/Forms/SearchBox/HxSearchBox.razor

```diff
@@ -67,8 +67,8 @@
 					string subtitle = ItemSubtitleSelector?.Invoke(item) ?? null;
 					IconBase icon = ItemIconSelector?.Invoke(item) ?? null;
 
-					<li class="overflow-hidden">
-						<a tabindex="0" class="@CssClassHelper.Combine("dropdown-item", ItemCssClassEffective)" @onclick="() => HandleItemSelected(item)">
+					<li class="overflow-hidden" @onkeypress="(eventArgs) => HandleKeyPress(eventArgs, async () => await HandleItemSelected(item))">
```

Comment:
@hakenr initially, we didn't need to handle this because the suggested items were rendered as buttons directly in the `HxDropdownMenu`. With structure changes introduced in [this commit](https://github.com/havit/Havit.Blazor/commit/ef89a4e7c0e3627b1657e39882c712b1cca6e3d1), we have to handle this functionality ourselves.

Reply:
(HxAutosuggest - same situation. We can set ItemTemplate there, but we kept button as a wrapper.)